docs/modules.md

```diff
@@ -133,7 +133,9 @@ var Module = fx.Module("server",
   ),
 )
 
-type Config struct {
+type Params struct {
+  fx.In
+
   Addr string `yaml:"addr"`
```

Comment:
Config isn't intended to be a parameter object. It is being passed into New through a Params struct.
Although I agree that this example is a tad confusing: Params and parseConfig aren't visible, and we just assume that a reader knows what they are.

The text in this example is generated automatically from ex/modules/module.go; it may require some adjustment in the snippets there to update this.